app/views/publishers/vacancies/_show.html.slim

```diff
@@ -0,0 +1,27 @@
+- content_for :page_title_prefix do
```

Comment:
I think the `do` here causes the bug where the page title contains a div tag. Can just be one line